src/client/Microsoft.Identity.Client/AuthScheme/PoP/MtlsPopAuthenticationOperation.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Identity.Client.Internal;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.AuthScheme.PoP
+{
+    internal class MtlsPopAuthenticationOperation : IAuthenticationOperation
+    {
+        private readonly X509Certificate2 _mtlsCert;
+
+        public MtlsPopAuthenticationOperation(X509Certificate2 mtlsCert)
+        {
+            _mtlsCert = mtlsCert;
+            KeyId = ComputeX5tS256KeyId(_mtlsCert);
+        }
+
+        public int TelemetryTokenType => (int)TokenType.Mtls_Pop;
```

Comment:
Currently we do not capture token type in client side telemetry. Is this something that we are looking to track through client side telemetry?

Reply:
The plan is to just track MTLS POP API usage at client side. Server side has the token type info. But is there other reasons why we should capture it on the client side?

Reply:
@trwalke for a perspective. I think this data is actually captured through the histograms.

Reply:
Will update once @trwalke merges the telemetry PR